OtripleS.Web.Api.Tests.Unit/Services/Foundations/Students/StudentServiceTests.Validations.Modify.cs

```diff
@@ -274,7 +273,6 @@ public async Task
             invalidStudent.CreatedBy = invalidCreatedBy;
             Guid studentId = invalidStudent.Id;
 
```

Comment:
You need to remove this line too.
Here's the C# Coding Standard for this:
https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations

Reply:
> You need to remove this line too. Here's the C# Coding Standard for this: https://github.com/hassanhabib/CSharpCodingStandard/blob/master/Variables.md#021-multiple-declarations

New Commit has been done as per instruction!